controller/config/beerocks_controller.conf.in

```diff
@@ -30,7 +30,7 @@ load_ire_roaming=1
 load_load_balancing=0
 
 #   Measurement feature:  
-load_diagnostics_measurements=0
+load_diagnostics_measurements=1
```

Comment:
Can you explain in the commit message what are the diagnostics measurements and why they are enabled now?

Reply:
I don't know why they enabled it. If I remember correctly you disabled it since it flooded the controller logs with timeout prints.
This task is responsible for periodically ask for statistics on hostapd and it's connected stations.
My guess is that the reason is to see `rx_rssi` value being updated in the connection map.
@vitalybu ?

Reply:
Please talk to Vitaly and add a comment so we can resolve this conversation.

Reply:
Ok, @vitalybu confirmed to me that he did it to update the rx_rssi. I'll add it to the commit message.

Reply:
updated the commit message.